### PR TITLE
Implement route to fetch user by ID

### LINKED ISSUE #7: Implement route to fetch user by ID
## Tasks
- Create controller to fetch user by ID
- Create route for GET request

### PR DESCRIPTION
**Pull request by aledeltoro**
Fixes #7 

### Dependencies
- [Implement route to delete user by ID](#18) ✅ 

## Proposed Changes
  - Create controller to fetch user by ID
  - Create route to fetch user by ID

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -7,10 +7,44 @@ const createUser = async (req, res) => {
     await user.save();
     res.status(201).json(user);
   } catch (err) {
-    res.status(400).send({status: 400, message: err.message});
+    res.status(400).send({ status: 400, message: err.message });
+  }
+};
+
+const fetchUserByID = async (req, res) => {
+  const _id = req.params.id;
```
Comment (cesarlopezt) [inline controllers/user.js]: Not sure why the underscore, but it looks good 😆

File: models/transaction.js [hunk 2]
```diff
@@ -46,6 +46,6 @@ const transactionSchema = new mongoose.Schema({
   },
 });
 
-const Transaction = mongoose.Model("transaction", transactionSchema);
+const Transaction = mongoose.model("transaction", transactionSchema);
```
Comment (cesarlopezt) [inline models/transaction.js:49]: 👍

**Review by cesarlopezt — APPROVED** [review]
Changes ready to be merged 👍

File: controllers/user.js (hunk 1 above)
Comment (aledeltoro) [reply controllers/user.js]: yeah, the naming is a bit odd. Fixed!

**Pull request merged**